ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -403,6 +406,22 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             true,
             ConfigDef.Importance.LOW,
             "Enable the INSERT INTO ... VALUES functionality."
+        ).define(
+            KSQL_PERSIST_SINGLE_FIELD_IN_STRUCT,
+            ConfigDef.Type.BOOLEAN,
+            true,
+            ConfigDef.Importance.LOW,
+            "Controls how KSQL will serialize a value whose schema contains only a "
```

Comment:
nit: might be nice to have this as a HELP constant without the newlines and then use `Splitter.fixedLength(80).splitToList(HELP).stream().collect(System.lineSeparator())`

Maybe even make a utility method out of that :)

Reply:
The new lines are more strategically placed that just at the 80 char boundary...

Reply:
fair enough